react-app/src/components/WalletConnectModal/WalletConnectModal.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { Fragment, useCallback } from "react";
+import { Dialog, Transition } from "@headlessui/react";
+import cn from "classnames";
+
+import { ReactComponent as KeplrLogo } from "../../assets/keplr-logo.svg";
+import { ReactComponent as WalletConnectLogo } from "../../assets/walletconnect-logo.svg";
+
+import Divider from "../common/Divider/Divider";
+import { Icon, IconType } from "../common/Icons/Icons";
+import LocalizedText from "../common/Localized/LocalizedText";
+
+interface WalletConnectModalProps {
+  isOpened: boolean;
+  onClose: () => void;
+}
+
+const WalletConnectModal: React.FC<WalletConnectModalProps> = (props) => {
```

Comment:
Might want to rename this to distinguish from the WalletConnect protocol; like `ConnectWalletModel`?